client/my-sites/customer-home/cards/tasks/single-task.jsx

```diff
@@ -0,0 +1,125 @@
+/**
+ * External dependencies
+ */
+import React, { useRef, useState } from 'react';
+import { useTranslate } from 'i18n-calypso';
+import { connect } from 'react-redux';
+import { Button } from '@automattic/components';
+import { isDesktop } from '@automattic/viewport';
+
+/**
+ * Internal dependencies
+ */
+import ActionPanel from 'components/action-panel';
+import ActionPanelTitle from 'components/action-panel/title';
+import ActionPanelBody from 'components/action-panel/body';
+import ActionPanelFigure from 'components/action-panel/figure';
+import ActionPanelCta from 'components/action-panel/cta';
+import Gridicon from 'components/gridicon';
+import PopoverMenu from 'components/popover/menu';
+import PopoverMenuItem from 'components/popover/menu-item';
+import { savePreference } from 'state/preferences/actions';
+import { getSelectedSiteId } from 'state/ui/selectors';
+
+/**
+ * Style dependencies
+ */
+import './style.scss';
+
+const SingleTask = ( {
+	title,
+	description,
+	actionText,
+	actionUrl,
+	illustration,
+	timing,
+	skipTask,
+} ) => {
+	const translate = useTranslate();
+	const [ isTaskVisible, setIsTaskVisible ] = useState( true );
+	const [ areSkipOptionsVisible, setSkipOptionsVisible ] = useState( false );
+	const skipButtonRef = useRef( null );
+
+	if ( ! isTaskVisible ) {
+		return null;
+	}
+
+	const remindLater = ( reminder ) => {
+		setIsTaskVisible( false );
+		skipTask( reminder );
+	};
+
+	return (
+		<ActionPanel className="single-task">
+			<ActionPanelBody>
+				{ isDesktop() && (
+					<ActionPanelFigure align="right">
+						<img src={ illustration } alt="" />
+					</ActionPanelFigure>
+				) }
+				<div className="single-task__timing">
+					<Gridicon icon="time" size={ 18 } />
+					<p>{ translate( '%d minute', '%d minutes', { count: timing, args: [ timing ] } ) }</p>
+				</div>
+				<ActionPanelTitle>{ title }</ActionPanelTitle>
+				<p className="single-task__description">{ description }</p>
+				<ActionPanelCta>
+					<Button className="single-task__action" primary href={ actionUrl }>
+						{ actionText }
+					</Button>
+					<Button
+						className="single-task__skip is-link"
+						ref={ skipButtonRef }
+						onClick={ () => {
+							setSkipOptionsVisible( true );
+						} }
+					>
+						{ translate( 'Remind me' ) }
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **52** times:
`translate( 'Remind Me Later' )` *ES Score: 8*
See 1 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/home-tasks-temporary-dismissal)